### PR TITLE
PCSX2: Update copyright headers to language translations (po files)

### LINKED ISSUE #2493: Locale files have unclear licensing
***Description of the issue:***
Two files have missing/unclear copyright and license statements. This makes them strictly speaking not redistributable under the GPL.  
These two files only have a standard template header:
- locales/es_ES/pcsx2_Iconized.po
- locales/es_ES/pcsx2_Main.po

The header is

    # SOME DESCRIPTIVE TITLE.
    # Copyright (C) YEAR PCSX2_Dev_Team
    # This file is distributed under the same license as the PACKAGE package.
    # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.

which looks like a template, that never got filled with the appropriate information about the author and the license (although PACKAGE can be guessed as PCSX2 by comparing it to the other locales).
Just to be on the safe side, these files should be correctly licensed.

### PR DESCRIPTION
**Pull request by lightningterror**
Close #2493

### CODE REVIEW

**Comment by lightningterror** [comment]
Not sure why travis keeps failing.

**Comment by turtleli** [comment]
> Not sure why travis keeps failing.

Maybe because whatever editor you used added some byte order marks?

**Comment by avih** [comment]
Why do we keep doing these massive updates? no other project does it, and it just adds noise. If you touch a file for real reason and the year has changed, maybe(!) consider updating the year too at this file, but otherwise, there's really no need.

File: locales/sv_SE/pcsx2_Iconized.po [hunk 1]
```diff
@@ -1,8 +1,10 @@
-# Copyright (C) 2011 PCSX2 Team
+﻿# Translations for PCSX2 (Iconized).
+# Copyright (C) 2018 PCSX2 Dev Team
 # This file is distributed under the same license as the PCSX2 package.
+#
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 - 180611\n"
+"Project-Id-Version: PCSX2 1.5.0\n"
```
[the remark this answers is not in the document]
Comment (lightningterror) [reply locales/sv_SE/pcsx2_Iconized.po]: The date should be added/changed in _PO-Revision-Date:_ not here.

**Comment by lightningterror** [comment]
Maybe we can drop _Project-Id-Version:_
I kinda don't see the point in keeping that, it just adds extra stuff to maintain.

**Comment by pgert** [comment]
Project-Id-Version is POedit standard.
Please revert and leave it to the translators.

**Comment by ssakash** [comment]
While I personally agree with @avih, I think an exception can be given for this as all the effort have already been put into it. But I'm future try to not add commits just for copyright updates, but rather update it only with functional changes on the file.

**Comment by lightningterror** [comment]
I updated the year since only a few had year changes but if you're fine with it I can leave out the year.
`Copyright (C) PCSX2 Dev Team`

I wanted to update the copyright headers because they were plain templates, and some that did have were not the same when comparing all of them, different wording.

**Comment by lightningterror** [comment]
Maybe I'll come back to this another time.

**Pull request closed without merging**